src/front/wgsl/mod.rs

```diff
@@ -2500,25 +2500,19 @@ impl Parser {
         self.push_scope(Scope::UnaryExpr, lexer);
         //TODO: refactor this to avoid backing up
         let expr = match lexer.peek().0 {
-            Token::Operation('-') => {
```

Comment:
The code for `'-'` was exactly the same as the one for `'!'` and `'~'` except for `op` so I thought refactoring a bit would make code easier to read.